### PR TITLE
add shib market

### PR DESCRIPTION
**Pull request by gmxer**

### CODE REVIEW

File: config/markets.ts [hunk 1]
```diff
@@ -702,6 +702,66 @@ const config: {
       thresholdForStableFunding: decimalToFloat(5, 2), // 5%
       thresholdForDecreaseFunding: decimalToFloat(0), // 0%
     },
+    {
+      tokens: { indexToken: "SHIB", longToken: "WETH", shortToken: "USDC" },
+      virtualTokenIdForIndexToken: hashString("PERP:SHIB/USD"),
+      virtualMarketId: hashString("SPOT:SHIB/USD"),
```
Comment (gdev8317) [inline config/markets.ts]: i think it should be SPOT:ETH/USD

looks like we have incorrect virtualMarketId for some older synthetic markets

File: config/markets.ts [hunk 2]
```diff
@@ -702,6 +702,66 @@ const config: {
...
+      reserveFactorLongs: percentageToFloat("80%"),
+      reserveFactorShorts: percentageToFloat("80%"),
+
+      openInterestReserveFactorLongs: percentageToFloat("75%"),
+      openInterestReserveFactorShorts: percentageToFloat("75%"),
+
+      maxPnlFactorForAdlLongs: percentageToFloat("50%"),
+      maxPnlFactorForAdlShorts: percentageToFloat("50%"),
+
+      // FIXME There's no Min Position size field, should be minPositionSize: decimalToFloat(1, 0), // 1
```
Comment (gdev8317) [inline config/markets.ts]: it's a global setting for all markets, no need to specify it here

File: config/markets.ts [hunk 3]
```diff
@@ -702,6 +702,66 @@ const config: {
...
+
+      // FIXME these fields duplicate baseMarketConfig defaults, remove them?
```
Comment (gdev8317) [inline config/markets.ts]: okay with me to remove it

File: config/markets.ts (hunk 1 above)
Comment (xvi10) [reply config/markets.ts]: agree it should be SPOT:ETH/USD

File: config/markets.ts (hunk 3 above)
Comment (xvi10) [reply config/markets.ts]: ok with me too

File: config/markets.ts [hunk 4]
```diff
@@ -702,6 +702,66 @@ const config: {
       thresholdForStableFunding: decimalToFloat(5, 2), // 5%
       thresholdForDecreaseFunding: decimalToFloat(0), // 0%
     },
+    {
+      tokens: { indexToken: "SHIB", longToken: "WETH", shortToken: "USDC" },
+      virtualTokenIdForIndexToken: hashString("PERP:SHIB/USD"),
+      virtualMarketId: hashString("SPOT:SHIB/USD"),
+
+      ...synthethicMarketConfig, // FIXME or synthethicMarketConfig_IncreasedCapacity ?
```
Comment (xvi10) [inline config/markets.ts]: i think it should be synthethicMarketConfig, cc @gdev8317

File: config/markets.ts [hunk 5]
```diff
@@ -702,6 +702,66 @@ const config: {
...
+      positivePositionImpactFactor: decimalToFloat(25, 11), // 0.05% for ~90,000 USD of imbalance
+
+      positiveMaxPositionImpactFactor: decimalToFloat(5, 3), // 0.5%
+      negativeMaxPositionImpactFactor: decimalToFloat(5, 3), // 0.5%
+
+      positiveSwapImpactFactor: decimalToFloat(25, 10), // 2.5e-9
+      negativeSwapImpactFactor: decimalToFloat(5, 9), // 5e-9
+
+      swapFeeFactorForPositiveImpact: decimalToFloat(5, 5), // 0.05%
+      swapFeeFactorForNegativeImpact: decimalToFloat(7, 5), // 0.07%
```
Comment (xvi10) [inline config/markets.ts]: could remove swapFeeFactorForPositiveImpact and swapFeeFactorForNegativeImpact, i don't think it is there in the other market configs

File: config/markets.ts [hunk 6]
```diff
@@ -702,6 +702,66 @@ const config: {
       thresholdForStableFunding: decimalToFloat(5, 2), // 5%
       thresholdForDecreaseFunding: decimalToFloat(0), // 0%
     },
+    {
+      tokens: { indexToken: "SHIB", longToken: "WETH", shortToken: "USDC" },
+      virtualTokenIdForIndexToken: hashString("PERP:SHIB/USD"),
+      virtualMarketId: hashString("SPOT:SHIB/USD"),
+
+      ...synthethicMarketConfig, // FIXME or synthethicMarketConfig_IncreasedCapacity ?
+
```
Comment (xvi10) [inline config/markets.ts:717]: missing

      maxLongTokenPoolAmount
      maxShortTokenPoolAmount

      maxLongTokenPoolUsdForDeposit
      maxShortTokenPoolUsdForDeposit

File: config/markets.ts (hunk 5 above)
Comment (gmxer) [reply config/markets.ts]: yep seems like it's only used in swapOnly markets

File: config/markets.ts (hunk 6 above)
Comment (gmxer) [reply config/markets.ts:717]: asked in chat for these

**Pull request merged**